data/css/meson-sassc

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2018 Endless Mobile, Inc.
+#
+# A small wrapper for sassc that generates a meson dependency file
+# for included partials.
+#
+# Usage: meson-sassc INPUT OUTPUT DEPFILE
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+
+_RE_IMPORT = re.compile(r'\s*@import\s+["]([A-Za-z0-9\-_]+)["]\s*')
+
+
+def matching_lines(compiled_regex, input_fileobj):
+    '''Yield only matches of compiled_regex.'''
+    for line in input_fileobj:
+        match = compiled_regex.match(line)
+        if match != None:
+            yield match.group(1)
+
+
+def compile_sass(arg_input, arg_output, arg_depfile):
+    '''Compile sass and write depfiles.'''
+    with open(arg_input, 'r') as input_fileobj:
```

Comment:
Not required by any means, but I remembered another way you can do this more conveniently: specify `type=argparse.FileType('r')` in the corresponding `add_argument` call, then `arguments.input` will already be an open file.